README.md

```diff
@@ -2,6 +2,17 @@
 Distributed system project
 
 1. [Basic architecture](#architecture)
+2. [Post/Get requests](#requests)
 
 ## Basic architecture <a name="architecture"></a>
 ![](https://github.com/AdoreIt/cv_masters_lockers/blob/master/architecture_diagram.png?raw=true)
+
+## Requests format <a name="requests"></a>
+###Get:
+	_Check_="name"           //_Check_ request
+	_Clear_="name"			 //_Clear_ request
+###Post:
+	_LockerBusy_="name"		 //All lockers are busy
+	_GetKey_="name":"key"	 //User got key
+	_Return_="name":"key"	 //User returned key
+	_Success_				 //_Clear_ action was successfull
```

Comment:
wrap code inside  ``` code block ``` 
with three ` sings before and after code block 
see here as a reference: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code